### PR TITLE
fix(builder-plugin-swc): correct corejs version

### PR DESCRIPTION
**Pull request by JSerFeng**
# PR Details

Fix wrong default corejs version

## Description

Corejs version config should be the same with @babel/preset-env

## Types of changes

- [ ] Docs change / refactoring / dependency upgrade
- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist

- [x] My code follows the code style of this project.
- [x] I have updated changeset
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: packages/toolkit/utils/package.json [hunk 1]
```diff
@@ -178,7 +178,8 @@
   },
   "dependencies": {
     "caniuse-lite": "^1.0.30001332",
-    "lodash": "^4.17.21"
+    "lodash": "^4.17.21",
+    "core-js": "^3.26.0"
```
Comment (chenjiahan) [inline packages/toolkit/utils/package.json:182]: core-js 这个依赖放在 utils 里面有点怪怪的，要不 CORE_JS_PATH 由上层传入吧，然后 core-js 还是安装在 swc 和 babel 相关的包里

Comment (JSerFeng) [reply packages/toolkit/utils/package.json:182]: 就是改版本的时候得一起改

Comment (chenjiahan) [reply packages/toolkit/utils/package.json:182]: 嗯嗯，这个没问题，仓库里后面会尽量保持依赖版本一致，也在考虑加一些检测工具

File: packages/toolkit/utils/src/getCoreJsVersion.ts [hunk 2]
```diff
@@ -1,10 +1,8 @@
 import * as fs from '../compiled/fs-extra';
 
-export const CORE_JS_PATH = require.resolve('core-js/package.json');
-
-export const getCoreJsVersion = () => {
+export const getCoreJsVersion = (corejs_pkg_path: string) => {
```
Comment (chenjiahan) [inline packages/toolkit/utils/src/getCoreJsVersion.ts:3]: rust 写多了 ^_^ 

JS 里变量命名统一用驼峰哈

Comment (JSerFeng) [reply packages/toolkit/utils/src/getCoreJsVersion.ts:3]: 完全是下意识的🤣

**Review by chenjiahan — APPROVED** [review]

**Comment by chenjiahan** [comment]
单测挂了哈

**Comment by JSerFeng** [comment]
Done

**Pull request merged**